delivery_deliverea/readme/INSTALL.rst

```diff
@@ -0,0 +1,8 @@
+This module needs the `unidecode` python library.
+
+It depends on the modules:
+
+* `base_delivery_carrier_label <https://github.com/OCA/delivery-carrier/tree/16.0/base_delivery_carrier_label>`_ repositorio OCA/delivery-carrier
```

Comment:
Remove this reference and the queue_job one.

Reply:
I removed this reference.